arch/xtensa/core/xtensa_intgen.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python3
+import re
+import fileinput
+
+# Pass an Xtensa core-isa.h file on stdin or the command line, emits a
```

Comment:
Another lying comment.  It takes the output of preprocessing the template file, not core-isa.h itself.